simple-fb/src/main/java/com/sromku/simple/fb/entities/PlaceTag.java

```diff
@@ -0,0 +1,107 @@
+package com.sromku.simple.fb.entities;
+
+
+import com.google.gson.annotations.SerializedName;
+
+public class PlaceTag {
+
+    private static final String ID = "id";
+    private static final String CREATED_TIME = "created_time";
+    private static final String PLACE = "place";
+
+    @SerializedName(ID)
+    private String mId;
+
+    @SerializedName(CREATED_TIME)
+    private String mCreatedTime;
+
+    @SerializedName(PLACE)
+    private Place mPlace;
+
+    public Place getPlace() {
+        return mPlace;
+    }
+
+    public String getCreatedTime() {
+        return mCreatedTime;
+    }
+
+    public String getId() {
+        return mId;
+    }
+
+
+    public class Place extends IdName{
```

Comment:
@xsorifc28 Please change the existing `Place` class and copy this inner class into existing one